Packs/SailPointIdentityIQ/Integrations/SailPointIdentityIQ/SailPointIdentityIQ.yml

```diff
@@ -0,0 +1,613 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: SailPointIdentityIQ
+  version: -1
+configuration:
+- additionalinfo: The FQDN/IP of the IdentityIQ server to connect
+  display: IdentityIQ Server URL (e.g. https://identityiq-server.com/identityiq)
+  name: identityiq_url
+  required: true
+  type: 0
+- defaultvalue: OAUTH
+  display: Authentication Type
+  hidden: false
+  name: auth_type
+  options:
+  - OAUTH
+  - BASIC
+  required: true
+  type: 15
+- additionalinfo: Client Id for OAuth 2.0
+  display: Client Id (for OAuth 2.0)
+  hidden: false
+  name: client_id
+  required: false
+  type: 4
+- additionalinfo: Client Secret for OAuth 2.0
+  display: Client Secret (for OAuth 2.0)
+  hidden: false
+  name: client_secret
+  required: false
+  type: 4
+- additionalinfo: Username for Basic Authentication
+  display: Username (for Basic Auth)
+  hidden: false
+  name: username
+  required: false
+  type: 0
+- additionalinfo: Password for Basic Authentication
+  display: Password (for Basic Auth)
+  hidden: false
+  name: password
+  required: false
+  type: 4
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: '50'
+  display: Maximum number of incidents per fetch
+  name: max_fetch
+  required: false
+  type: 0
+- defaultvalue: 3 days
+  display: First fetch time
+  name: first_fetch
```

Comment:
This parameter is not being used - you always go back 3 days back in the first fetch.
Please edit the code to take this parameter into account.

Reply:
This was omitted due to dateparser issue that I mentioned above, but is now resolved and is being used.